code/stable/gamephysics/SRS/HTML/GamePhysics_SRS.html

```diff
@@ -2651,7 +2651,7 @@ <h2>Non-Functional Requirements</h2>
               </p>
               <p>
                 <div id="maintainability">
-                  Maintainability: The development time for any of the likely changes should not exceed 10<em>%</em> percent of the original development time.
+                  Maintainability: If a likely change is made to the finished software, it will take 10<em>%</em> percent of the original development time, assuming the same development resources are available.
```

Comment:
"10% percent" is redundant; it should probably just be "10%"

Also, is there a reason that the percent sign is in italics? I know this isn't part of this PR, so this might be pulled out to a new issue to at least investigate (but I think it shouldn't be in italics).

Reply:
Looks like this has been moved to #3748